benches/benches/db_lookup_times_utils/seed.rs

```diff
@@ -0,0 +1,151 @@
+use crate::db_lookup_times_utils::{
+    matrix::matrix,
+    utils::{
+        chain_id,
+        open_raw_rocksdb,
+    },
+};
+use fuel_core::{
+    database::database_description::on_chain::OnChain,
+    state::rocks_db::RocksDb,
+};
+use fuel_core_storage::{
+    column::Column,
+    kv_store::{
+        KeyValueMutate,
+        Value,
+    },
+};
+use fuel_core_types::{
+    blockchain::{
+        block::{
+            Block,
+            PartialFuelBlock,
+        },
+        header::{
+            ConsensusHeader,
+            PartialBlockHeader,
+        },
+        primitives::Empty,
+    },
+    fuel_tx::{
+        Transaction,
+        UniqueIdentifier,
+    },
+    fuel_types::BlockHeight,
+};
+
+pub fn seed_compressed_blocks_and_transactions_matrix(method: &str) {
+    for (block_count, tx_count) in matrix() {
+        let mut database = open_raw_rocksdb(block_count, tx_count, method);
+        let _ =
+            seed_compressed_blocks_and_transactions(&mut database, block_count, tx_count);
+    }
+}
+
+pub fn seed_full_block_matrix() {
+    for (block_count, tx_count) in matrix() {
+        let mut database = open_raw_rocksdb(block_count, tx_count, "full_block");
+        seed_full_blocks(&mut database, block_count, tx_count);
+    }
+}
+
+fn generate_bench_block(height: u32, tx_count: u32) -> Block {
+    let header = PartialBlockHeader {
+        application: Default::default(),
+        consensus: ConsensusHeader::<Empty> {
+            height: BlockHeight::from(height),
+            ..Default::default()
+        },
+    };
+    let txes = generate_bench_transactions(tx_count);
+    let block = PartialFuelBlock::new(header, txes);
+    block.generate(&[], Default::default()).unwrap()
+}
+
+fn generate_bench_transactions(tx_count: u32) -> Vec<Transaction> {
+    let mut txes = vec![];
+    for _ in 0..tx_count {
+        txes.push(Transaction::default_test_tx());
```

Comment:
[nit] It would be nice to make transactions random to be sure that compression will not affect it too much

Reply:
under the hood the transactions are randomized ~ not sure if I understand correctly!
<img width="584" alt="image" src="https://github.com/user-attachments/assets/1e79ca31-2f33-4132-95a6-65b19cb17186">

Reply:
`add_random_fee_input` always return the same result=D

Reply:
lol, okay ~ will alter